src/ckb/contracts.rs

```diff
@@ -1,456 +1,207 @@
-                        DepType::Code,
-                        DepType::DepGroup,
-                        vec![Contract::FundingLock, Contract::CommitmentLock],
-                    ),
-                    (DepType::Code, DepType::Code, vec![Contract::SimpleUDT]),
-                ] {
-                    for contract in contracts {
-                        let program_code_hash = get_hash_from_environment_variable(
-                            contract,
-                            EnvironmentVariableType::CodeHash,
-                            program_dep_type,
-                        );
-                        let group_tx = get_hash_from_environment_variable(
-                            contract,
-                            EnvironmentVariableType::TxHash,
-                            group_dep_type,
-                        );
-                        let group_index: usize = get_environment_variable(
-                            contract,
-                            EnvironmentVariableType::TxIndex,
-                            group_dep_type,
-                        )
-                        .parse()
-                        .expect("Valid index");
-                        let dep_group_out_point = OutPoint::new_builder()
-                            .tx_hash(group_tx.into())
-                            .index(group_index.pack())
-                            .build();
-                        let script = Script::new_builder()
-                            .code_hash(program_code_hash.into())
-                            .hash_type(ScriptHashType::Data1.into())
-                            .args(Bytes::new().pack())
-                            .build();
-                        let cell_dep = CellDep::new_builder()
-                            .out_point(dep_group_out_point.clone())
-                            .dep_type(group_dep_type.into())
-                            .build();
-                        map.insert(contract, script);
-                        script_cell_deps.insert(
-                            contract,
-                            CellDepVec::new_builder().push(cell_dep).build().pack(),
-                        );
-                    }
-                }
-                let tx1_env_name = format!("{ENV_PREFIX}_CKB_GENESIS_TX_1");
-                let tx1 = Hash256::from_str(
-                    &env::var(&tx1_env_name)
-                        .unwrap_or_else(|_| panic!("environment variable {tx1_env_name}")),
-                )
-                .expect("valid hash");
+            "0x10639e0895502b5688a6be8cf69460d76541bfa4821629d86d62ba0aae3f9606" => {
+                info!("Creating ContractsContext for testnet");
+            }
+            _ => {
+                info!("Creating ContractsContext for dev");
+                // index from 5 ~ 8 are the default contracts: CkbAuth, FundingLock, CommitmentLock, SimpleUDT
+                let ckb_auth_cell_dep = CellDep::new_builder()
```

Comment:
it seems better with this trivial refactor: https://github.com/quake/cfn-node/pull/5